src/components/molecules/BenefitFilter.js

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { ActionButton } from "../atoms/ActionButton";
+import { FilteredBenefitsCounter } from "../atoms/FilteredBenefitsCounter";
+
+/**
+ * This component displays three navigation buttons which filters out the result of benefits
+ */
+export function BenefitFilter(props) {
+  const handleClick = () => props.filter();
+
+  return (
+    <div>
+      <p className="text-gray-700 leading-none font-bold">{props.text}</p>
+      <div className="w-full md:flex">
+        <span>
+          <ActionButton
+            text={props.eligible}
+            invert={props.isSelectedEligible === false}
+            onClick={handleClick}
+          >
+            <FilteredBenefitsCounter count={props.eligibleCount} />
+          </ActionButton>
+        </span>
+        <span className="md:ml-2">
+          <ActionButton
+            text={props.help}
+            invert={props.isSelectedHelp === false}
+            onClick={handleClick}
+          >
+            <FilteredBenefitsCounter count={props.helpCount} />
+          </ActionButton>
+        </span>
+        <span className="md:ml-2">
+          <ActionButton
+            text={props.others}
+            invert={props.isSelectedOthers === false}
+            onClick={handleClick}
+          >
+            <FilteredBenefitsCounter count={props.othersCount} />
+          </ActionButton>
+        </span>
+      </div>
+    </div>
+  );
+}
+
+BenefitFilter.propTypes = {
+  /**
+   * Text on top of the buttons
+   */
+  text: PropTypes.string,
+
+  /**
+   * Button text that displays Eligible benefits
+   */
+  eligible: PropTypes.string,
+
+  /**
+   * Displays the number of results of eligible benefits
+   */
+  eligibleCount: PropTypes.number,
+
+  /**
+   * Button text that displays Potential help
+   */
+  help: PropTypes.string,
+
+  /**
+   * Displays the number of results of potential help
+   */
+  helpCount: PropTypes.number,
+
+  /**
+   * Other filter options that may be applied
+   */
+  others: PropTypes.string,
+
+  /**
+   * Display the number of results of other option that may applies
+   */
+  othersCount: PropTypes.number,
+
+  /**
+   * Handler when filter options clicked
+   */
+  filter: PropTypes.func,
```

Comment:
event callbacks should start with the prefix `on`. So this should be called `onFilter`. You'll notice in storybooks if you click the buttons nothing appears in the Actions tab. 

![Screen Shot 2020-10-29 at 10 20 12 AM](https://user-images.githubusercontent.com/26819992/97586166-55e69980-19d0-11eb-931b-dee7a80e3c06.png)

As opposed to something like the benefits card. Where when I click on the button I see the callback being called
![Screen Shot 2020-10-29 at 10 20 44 AM](https://user-images.githubusercontent.com/26819992/97586244-69920000-19d0-11eb-8f3d-62782fa38714.png)